rsudp/c_rsam.py

```diff
@@ -14,23 +15,28 @@ class RSAM(rs.ConsumerThread):
 	"""
 	.. versionadded:: 1.0.0
 
-	A consumer class that runs an Real-time Seismic Analysis Measurement (RSAM).
+	A consumer class that runs an Real-time Seismic Amplitude Measurement (RSAM).
 
 	:param queue.Queue q: queue of data and messages sent by :class:`rsudp.c_consumer.Consumer`.
 	:param bool debug: whether or not to display RSAM analysis live to the console.
 	:param float interval: window of time in seconds to apply RSAM analysis.
 	:param str cha: listening channel (defaults to [S,E]HZ)
 	:param str deconv: ``'VEL'``, ``'ACC'``, ``'GRAV'``, ``'DISP'``, or ``'CHAN'``
+	:param str fwaddr: Specify a forwarding address to send RSAM in a UDP packet
+	:param str fwport: Specify a forwarding port to send RSAM in a UDP packet
 	"""
 
-	def __init__(self, q=False, debug=False, interval=5, cha='HZ', deconv=False, *args, **kwargs):
+	def __init__(self, q=False, debug=False, interval=5, cha='HZ', deconv=False, fwaddr=False, fwport=False, *args, **kwargs):
 		"""
 		Initializes the RSAM analysis thread.
 		"""
 		super().__init__()
 		self.sender = 'RSAM'
 		self.alive = True
 		self.debug = debug
+		self.fwaddr = fwaddr
+		self.fwport = fwport
+		self.sock = False
 		self.interval = interval
 		self.default_ch = 'HZ'
```

Comment:
```
self.stn = rs.stn

Reply:
I got a formatting error when attempting this:
```
Traceback (most recent call last):
  File "/home/justin/miniconda3/envs/rsudp/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/home/justin/miniconda3/envs/rsudp/lib/python3.7/site-packages/rsudp/c_rsam.py", line 253, in run
    (self.interval, self.stn, self.cha, self.fwaddr),
TypeError: not all arguments converted during string formatting
```

Is that an object and not a string?

Reply:
Nevermind looks like a code issue on my end :)